lib/http/request.js

```diff
@@ -249,7 +249,7 @@ class HttpRequest extends EventEmitter {
       });
     }
 
-    const content = `  Request ${Logger.colors.light_cyan([this.reqOptions.method, this.hostname + this.reqOptions.path, retryStr + ' '].join(' '))}`;
+    const content = `  Request ${[this.reqOptions.method, this.hostname + this.reqOptions.path, retryStr + ' '].join(' ')}`;
```

Comment:
That would be added [here](https://github.com/nightwatchjs/nightwatch/pull/3852/files#diff-1a88f23636217e552be27dabca2f7000f06bed95c17a8babe14443cf6b5df56aR95). Color should either be added to the whole content (including 'Request', otherwise it would be left uncolored), or not at all (in which case it will pick the color specified for `Logger.info`.